core-java-modules/core-java-io-apis-2/pom.xml

```diff
@@ -81,6 +81,12 @@
             <version>7.1.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.4.1</version>
+            <scope>test</scope>
+        </dependency>

```

Comment:
This is already included by the parent pom.xml, so we can remove it. Or if you want to keep it, we have a convention to extract a property for the version number. The version used in the parent is 3.21.0.

Reply:
Removed.